apis/system/v1alpha1/system.go

```diff
@@ -97,3 +99,26 @@ type SystemList struct {
 	metav1.ListMeta `json:"metadata,omitempty"`
 	Items           []System `json:"items"`
 }
+
+// HasLabels whether the system has labels
+func (in SystemParameters) HasLabels() bool {
```

Comment:
I think you you can use a pointer in all of these methods since there is not need for call-by-value.